EventsExpress.Core/Services/EventService.cs

```diff
@@ -292,7 +292,8 @@ public async Task<Guid> Publish(Guid eventId)
                         UserId = CurrentUser().Id,
                     });
                 await Context.SaveChangesAsync();
-
+                EventDto dtos = Mapper.Map<Event, EventDto>(ev);
+                await _mediator.Publish(new EventCreatedMessage(dtos));
```

Comment:
Let add a unit test to verify this method was invoked to avoid the same issue in the future